### ISSUE TITLE
facing fatal error repeatedly after upgrade to 1.7.3

### ISSUE THREAD

**Issue by elsonr** [osTicket/osTicket-1.7#860]
Hi,

I am getting fatal error soon after I have upgraded to version 1.7.3.
I am running 1.7.3 on windows 2008 with php 5.4.19.

Soon after I have upgraded to v1.7.3 the system worked smoothly for a while and now it has started throwing this error. It  becomes normal after a while but this error appears again and again.

when I looked into the logs I found the below error. Please help me solve this issue.
Log Name:      Application
Source:        PHP-5.4.19
Date:          11/30/2013 6:01:49 PM
Event ID:      2
Task Category: (6)
Level:         Information
Keywords:      Classic
User:          N/A
Computer:      WEBAPP
Description:
The description for Event ID 2 from source PHP-5.4.19 cannot be found. Either the component that raises this event is not installed on your local computer or the installation is corrupted. You can install or repair the component on the local computer.

If the event originated on another computer, the display information had to be saved with the event.

The following information was included with the event: 

php[6032]
PHP Warning:  mail(): SMTP server response: 530 Authentication required to send mail from  in D:\support.vakrangee.in\upload\include\pear\Mail\mail.php on line 153

Event Xml:
<Event xmlns="http://schemas.microsoft.com/win/2004/08/events/event">
  <System>
    <Provider Name="PHP-5.4.19" />
    <EventID Qualifiers="16384">2</EventID>
    <Level>4</Level>
    <Task>6</Task>
    <Keywords>0x80000000000000</Keywords>
    <TimeCreated SystemTime="2013-11-30T12:31:49.000000000Z" />
    <EventRecordID>333822</EventRecordID>
    <Channel>Application</Channel>
    <Computer>WEBAPP</Computer>
    <Security />
  </System>
  <EventData>
    <Data>php[6032]</Data>
    <Data>PHP Warning:  mail(): SMTP server response: 530 Authentication required to send mail from  in D:\URL\upload\include\pear\Mail\mail.php on line 153</Data>
  </EventData>
</Event>

**Comment by elsonr** [comment]
Sometimes it is also throwing HTTP 500 Internal Server Error. 

Please help me solve this issue

Regards,
Elson

**Comment by greezybacon** [comment]
Do you have authentication enabled for the SMTP side of the mail account you are sending through?

**Comment by elsonr** [comment]
yes the authentication is enabled and the credentials used are working as tested in osTicket using Diagnostic option.

**Comment by greezybacon** [comment]
Sorry, but I just have to make sure, did you select the "From" account in question when sending the email from the diagnostic page?

**Comment by elsonr** [comment]
Yes I had selected the "From" account and the mail is successfully sent to the Email Address added in "TO".
